storage/shard.go

```diff
@@ -424,9 +427,9 @@ func (s *dbShard) Bootstrap(
 
 	multiErr := xerrors.NewMultiError()
 	for _, dbBlocks := range bootstrappedSeries {
-		series, _, completionFn := s.writableSeries(dbBlocks.ID)
-		err := series.Bootstrap(dbBlocks.Blocks)
-		completionFn()
+		entry := s.writableSeries(dbBlocks.ID)
+		err := entry.series.Bootstrap(dbBlocks.Blocks)
+		entry.decrementWriterCount()
```

Comment:
Agreed, maybe we should just rename the the existing methods to make it clear what they're doing.